mcstatus/pinger.py

```diff
@@ -139,8 +139,13 @@ def __init__(self, raw):
 
         if "description" not in raw:
             raise ValueError("Invalid status object (no 'description' value)")
-        self.description = raw["description"]
-        self.description_clean = re.sub(r'\u00A7.', '', raw["description"])
+        if "text" not in raw["description"]:
```

Comment:
Wouldn't this attempt to subscript the value if raw["description"] was a string that contained the word "text"? I think an explicit type check may be safer in this case